configure.ac

```diff
@@ -4115,6 +4115,60 @@ AC_SUBST(HAVE_GETHOSTBYNAME_R_3_ARG)
 AC_SUBST(HAVE_GETHOSTBYNAME_R)
 AC_SUBST(HAVE_GETHOSTBYNAME)
 
+# checks for re-entrant nss functions
+AC_CHECK_FUNCS(getpwnam_r,
```

Comment:
Why not just adding the check to the existing long "# checks for library functions
AC_CHECK_FUNCS(alarm accept4 setitimer getitimer bind_textdomain_codeset chown ..." list?

Reply:
Yes, sorry, I am not too familiar with autotools, will change to use that instead, thanks for the pointer.